docs/API.MD

```diff
@@ -35,14 +38,19 @@ export default class Home extends Component {
 }
 ```
 
-The interface is similar to react-redux connect. The keys of this object will be used to connect data returned from from promise to corresponding prop in the component.
+The interface is similar to react-redux connect. The key of each asyncItem object will be used to connect data returned from from promise to corresponding prop in the component.
 So in example above you'll have this.props.lunches
 
-The value of each key should be function, that accepts params from router and helpers. Redux store exists in helpers by default.
-This function can return:
+The value of each ayncItem 'key' should be a string identifier which will become a key on reduxAsyncConnect redux state tree.
+The value of each asyncItem 'promise' should *usually be a promise* that accepts params from router and helpers
+
```

Comment:
Lets look at this from the server perspective:

1. `loadOnServer(Object)` : http://github.com/makeomatic/redux-connect/blob/a40386f83473918b35daba50b1b497ca5e6e5b40/modules/helpers/utils.js#L94
2. Goes to `loadAsyncConnect({ components, filter = () => true, ...rest })`
3. Every asyncItem is called with `let promiseOrResult = item.promise(rest);` - notice `rest` from step (2)

`rest` includes `store` and everything else that you've passed, like `helpers` and router props